styles.css

```diff
@@ -139,3 +141,188 @@ h1 {
   border-radius: 50%;
   padding: 2%;
 }
+
+h2 {
+  font-family: "Inter", sans-serif;
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+  color: #344563;
+  margin-top: 2%;
+}
+
+.h3_my_work {
+  font-family: "Inter", sans-serif;
+  text-align: center;
+  font-size: 40px;
+  color: #172b4d;
+  font-weight: 700;
+  margin-top: 15%;
+}
+
+.work_card_container {
+  margin-block: 60px;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 24px;
+}
+
+.img_holder {
+  display: inline;
+  height: 53%;
+  background-color: #d0d9d4;
+}
+
+.project_card {
+  height: 55vh;
+  background-color: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  margin: 0% 4%;
+}
+
+.large_title {
+  font-family: "Inter", sans-serif;
+  text-align: center;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  margin-top: 5%;
+  color: #3a4a42;
+}
+
+.tools_flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 0;
+  gap: 8px;
+}
+
+.skills_design {
+  height: 30px;
+  border: none;
+  border-radius: 4px;
+  font-weight: 600;
+  font-size: 12px;
+  padding: 10px 12px;
+  color: #36b37e;
+  background-color: #fff;
+}
+
+.tools_design {
+  height: 30px;
+  border: none;
+  border-radius: 4px;
+  color: #3a4a42;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  letter-spacing: 0.02cm;
+  margin: auto;
+  padding: 10px 12px;
+}
+
+.btngreen {
+  align-self: center;
+  width: 150px;
+  height: 50px;
+  font-weight: 500;
+  font-size: 17px;
+  background-color: #36b37e;
+  color: #fff;
+  margin-top: 5%;
+  border: none;
+  border-radius: 4px;
+  margin-bottom: 5%;
+}
+
+.btngreen:hover {
+  border-radius: 4px;
+  background-color: #36b37e;
+}
```

Comment:
- [ ] ***[OPTIONAL]*** You have used the same `background-color: #36b37e`, for both while `hovering` and `not-hovering`. But it shouldn't be like this. Kindly fix this issue.
- Kindly check [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=BgvM3581QXUlMIdm-0) for details.
![image](https://user-images.githubusercontent.com/60083437/236548319-25a5ca86-e47e-4559-9382-c24228e63d36.png)